src/display/dom_utils.js

```diff
@@ -33,6 +33,37 @@ var createValidAbsoluteUrl = sharedUtil.createValidAbsoluteUrl;
 
 var DEFAULT_LINK_REL = 'noopener noreferrer nofollow';
 
+function DOMCanvasFactory() {}
+DOMCanvasFactory.prototype = {
+  create: function DOMCanvasFactory_create(width, height) {
+    var canvas = document.createElement('canvas');
+    canvas.width = width || 0;
+    canvas.height = height || 0;
```

Comment:
actually, the code at isFontSubpixelAAEnabled was just creating canvas with default size, we just need to remove `|| 0` or `typeof width !== 'undefined'` and specify needed canvas size at the caller (isFontSubpixelAAEnabled) e.g. 200x100 or 50x10